### ISSUE TITLE
gnu(9920)=12142

### ISSUE THREAD

**Issue by olexandr-konovalov** [alex-konovalov/gnu#35]
After 6 hours on Linux server (AMD Opteron 6376 1400 MHz processor, 512 GB RAM) `grpconst.sh 9920` produced the following:

```
GAP      4.8.3
GrpConst 2.5
CubeFree 1.15
SglPPow  1.1
Runtime: 21326384 ms
Constructed a list of length 12141
but can not check that it is irredundant
```

One should rerun it (after #27) to spilt lists of groups contained in its output.

**Comment by olexandr-konovalov** [comment]
gnu(9920)=12142

GAP      4.8.3
GrpConst 2.5
Runtime of ConstructAllGroups: 14035496 ms
Returned a list of length 12141 containing 12140 groups and 1 sublists
Length of sublists : [ 2 ]
Runtime to check non-isomorphism of groups in pairs: 200249040

Replicated by @alex-konovalov on Ubuntu 16.04 on Azure cloud standard DS13 v2 instance (8 cores, 56 GB memory)

In case this value is new, add the next line to data/gnudata.g
GNU_SAVE( 9920, 12142, WITH_GC );

**Comment by olexandr-konovalov** [comment]
Closed as duplicate of #67.
